.github/workflows/integrate.yaml

```diff
@@ -44,23 +44,12 @@ jobs:
     - uses: balchua/microk8s-actions@v0.2.2
```

Comment:
Maybe you want to use `charmed-kubernetes/actions-operator@main` instead and save a lot of boilerplate down below, like selecting the charmcraft channel to use

Reply:
I agree that we want to that, but I leave these tasks available for onboarders to convert them.